RadixWallet/Features/TransactionReviewFeature/TransactionReview+View.swift

```diff
@@ -26,13 +26,15 @@ extension TransactionReview.State {
 				return nil
 			}(),
 			isExpandedDappUsed: dAppsUsed?.isExpanded == true,
+			isExpandedContributingToPools: contributingToPools?.isExpanded == true,
+			isExpandedRedeemingFromPools: redeemingFromPools?.isExpanded == true,
 			showTransferLine: withdrawals != nil && deposits != nil,
 			viewControlState: viewControlState,
 			rawTransaction: displayMode.rawTransaction,
 			showApprovalSlider: reviewedTransaction != nil,
 			canApproveTX: canApproveTX && reviewedTransaction?.feePayingValidation.wrappedValue == .valid,
 			sliderResetDate: sliderResetDate,
-			canToggleViewMode: reviewedTransaction != nil && reviewedTransaction?.transaction != .nonConforming,
+			canToggleViewMode: reviewedTransaction != nil && reviewedTransaction?.isNonConforming != false,
```

Comment:
is `reviewedTransaction?.isNonConforming != false` correct?  shouldn't this be `reviewedTransaction?.isNonConforming == false`?

Reply:
Yes you are right, thanks